src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/validation/EnvelopeProcessorValidationTest.java

```diff
@@ -11,24 +18,55 @@
 import uk.gov.hmcts.reform.bulkscanprocessor.model.ocr.OcrData;
 import uk.gov.hmcts.reform.bulkscanprocessor.model.ocr.OcrDataField;
 import uk.gov.hmcts.reform.bulkscanprocessor.services.document.output.Pdf;
+import uk.gov.hmcts.reform.bulkscanprocessor.tasks.processor.EnvelopeProcessor;
 
 import java.util.List;
 
 import static java.util.Arrays.asList;
+import static java.util.Collections.emptyList;
 import static java.util.Collections.singletonList;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.catchThrowable;
+import static org.mockito.Mockito.when;
 import static uk.gov.hmcts.reform.bulkscanprocessor.helper.InputEnvelopeCreator.inputEnvelope;
 import static uk.gov.hmcts.reform.bulkscanprocessor.helper.InputEnvelopeCreator.scannableItem;
 
 @SuppressWarnings("checkstyle:LineLength")
+@RunWith(MockitoJUnitRunner.class)
 public class EnvelopeProcessorValidationTest {
 
+    private EnvelopeProcessor envelopeProcessor;
```

Comment:
weird, this test did not test this class before. is this the right place?

Reply:
We don't have an existing unit tests for `EnvelopeProcessor`. I thought this is the right place since the class name includes `EnmvelopeProcessor`

Reply:
it contains tests for `EnvelopeValidator`

Reply:
(which is where we should also validate the po box)

Reply:
Moved validation to `EnvelopeValidator`. so this test no longer have references to `EnvelopeProcessor`